crypto/evp/p_lib.c

```diff
@@ -1316,30 +1316,32 @@ int EVP_PKEY_supports_digest_nid(EVP_PKEY *pkey, int nid)
     return rv;
 }
 
-int EVP_PKEY_set1_tls_encodedpoint(EVP_PKEY *pkey,
-                               const unsigned char *pt, size_t ptlen)
+int EVP_PKEY_set1_encoded_public_key(EVP_PKEY *pkey, const unsigned char *pub,
+                                     size_t publen)
 {
     if (pkey->ameth == NULL) {
-        OSSL_PARAM params[2] = { OSSL_PARAM_END, OSSL_PARAM_END };
+        OSSL_PARAM params[2]
+            = { OSSL_PARAM_END, OSSL_PARAM_END };
```

Comment:
Nit: Why was this line break necessary?

Reply:
It wasn't this is an artefact from an earlier iteration of development - so this change is no longer necessary and I have removed it.